README.md

```diff
@@ -4,17 +4,116 @@ This is a service used within the NHS for managing and recording school-aged vac
-| [Training](https://github.com/nhsuk/manage-vaccinations-in-schools/deployments/training)     | [training.manage-vaccinations-in-schools.nhs.uk](https://training.manage-vaccinations-in-schools.nhs.uk) | External training            | ❌                  | release | [`staging`](config/environments/staging.rb)       |
-| [Production](https://github.com/nhsuk/manage-vaccinations-in-schools/deployments/production) | [www.manage-vaccinations-in-schools.nhs.uk](https://www.manage-vaccinations-in-schools.nhs.uk)           | Live service                 | ✅                  | release | [`production`](config/environments/production.rb) |
+| Name                                                                                         | URL                                                                                                      | Purpose                      | Care Identity login | Code                   | Deployment | `RAILS_ENV`                                       |
+| -------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- | ---------------------------- | ------------------- | ---------------------- | ---------- | ------------------------------------------------- |
+| [Heroku](https://github.com/nhsuk/manage-vaccinations-in-schools/deployments/heroku)         | mavis-pr-xxxx.herokuapp.com                                                                              | PR review apps               | ❌                  | `main` branch (latest) | automated  | [`staging`](config/environments/staging.rb)       |
+| [Test](https://github.com/nhsuk/manage-vaccinations-in-schools/deployments/test)             | [test.mavistesting.com](https://test.mavistesting.com)                                                   | Internal testing (manual)    | ✅                  | `main` branch (latest) | automated  | [`staging`](config/environments/staging.rb)       |
+| [QA](https://github.com/nhsuk/manage-vaccinations-in-schools/deployments/qa)                 | [qa.mavistesting.com](https://qa.mavistesting.com)                                                       | Internal testing (automated) | ❌                  | `main` branch (latest) | automated  | [`staging`](config/environments/staging.rb)       |
+| [Preview](https://github.com/nhsuk/manage-vaccinations-in-schools/deployments/Preview)       | [preview.mavistesting.com](https://preview.mavistesting.com)                                             | External testing             | ❌                  | `release` branch       | manual     | [`staging`](config/environments/staging.rb)       |
```

Comment:
`preview` won't always be on `release`, because we'll be deploying the next release candidate to allow the SAIS teams to do testing before going live in production.